src/Illuminate/Mail/Mailer.php

```diff
@@ -6,14 +6,14 @@
 use Swift_Message;
 use Illuminate\Support\Arr;
 use InvalidArgumentException;
-use Illuminate\Support\HtmlString;
 use Illuminate\Contracts\View\Factory;
 use Illuminate\Contracts\Events\Dispatcher;
 use Illuminate\Contracts\Queue\ShouldQueue;
 use Illuminate\Contracts\Mail\Mailer as MailerContract;
 use Illuminate\Contracts\Queue\Factory as QueueContract;
 use Illuminate\Contracts\Mail\Mailable as MailableContract;
 use Illuminate\Contracts\Mail\MailQueue as MailQueueContract;
+use Illuminate\Contracts\Support\Htmlable as HtmlableContract;
```

Comment:
There's no need to alias this. The other have been aliases due to conflicts or ambiguity.